main.ts

```diff
@@ -55,7 +55,14 @@ bluetooth.onUartDataReceived(serial.delimiters(Delimiters.NewLine), () => {
       bluetooth.uartReadUntil(serial.delimiters(Delimiters.NewLine))
     );
     gigglebot.motorPowerAssign(gigglebotWhichMotor.Right, value);
+    led.plotBrightness(4, 0, (value * 255) / 100);
+  } else if ("both-motors" === command) {
+    value = parseFloat(
```

Comment:
Since this is done in all three cases, you could pull it out of the if statement and just have `value` for setting the proper motor later

Reply:
That's true. I was thinking that future commands in the protocol may not have a value, so I was wanting to keep it more general.